.changelog/unreleased/bug-fixes/3270-no-panic-voteext.md

```diff
@@ -0,0 +1,2 @@
+- `[cometbft]` Prevent panic on non-nil Precommit when vote extensions are enabled
+  cometbft ([\#3270](https://github.com/cosmos/gaia/pull/3270)
```

Comment:
This PR bumps the version of CometBFT. The bug was fixed in the comet repo and it's marked in their changelogs. 
```suggestion
- Bump [cometbft](https://github.com/cometbft/cometbft) to
   [v0.38.11](https://github.com/cometbft/cometbft/releases/tag/v0.38.11)
   ([\#3270](https://github.com/cosmos/gaia/pull/3270)
```